README.md

```diff
@@ -4,3 +4,53 @@
 
 To quickly setup this project run `make init`. To get more help, simply run
 `make help`.
+
+
+## Key build operations
+
+This project uses Maven as its build system. Unless stated otherwise, instructions below assume that Maven commands
+will be executed from the directory of the parent module.
+
+To run:
+
+**Unit tests**:
+```
+mvn test
+```
+
+**Full suite of tests**:
```

Comment:
Cosmetic thing, really, but this being vanilla version of the project, we're better off loosing the 'Full suite of tests' section as this version of the project does not include any acceptance tests module.